olcPGEX_AnimatedSprite.h

```diff
@@ -98,8 +99,9 @@ namespace olc
 		std::string state;
 		std::map<std::string, std::vector<olc::Sprite*>> multiFrames;
 		std::map<std::string, std::vector<olc::vi2d>> singleFrames;
+		std::map<std::string, float> frameDurations;
 		float frameTimer = 0.0f;
-		float frameDuration = 0.1f;
+		//float frameDuration = 0.1f;
```

Comment:
I think this should be kept as a default value (see above comment).

Reply:
Added it back, this time as a constant.